packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/types/ImageHtmlOptions.ts

```diff
@@ -19,4 +21,22 @@ export default interface ImageHtmlOptions {
      * Background color of the rotate handle
      */
     rotateHandleBackColor: string;
+
+    /**
+     * Image editing data
+     */
+    editInfo: ImageEditInfo;
+
+    /**
+     * The number of handlers should be adapted to the image size
+     * 8 handlers: 100x100px or bigger
+     * 4 handlers: 50x50px or bigger
+     * 1 handler: smaller than 50x50px
+     */
+    sizeAdaptiveHandlers?: boolean;
+
+    /**
+     * The handlers should have circular borders
+     */
+    circularHandlers?: boolean;
```

Comment:
I think we don't need this one. Let's just change it to circular if we have decided

Reply:
I think we can use this property to make a flight for this feature.